src/test/java/com/easypost/BatchTest.java

```diff
@@ -154,6 +190,7 @@ public void testAddRemoveShipment() throws EasyPostException {
         shipmentData.add(shipment);
         params.put("shipments", shipmentData);
 
+        // TODO: We shouldn't be requiring users to put the list of shipments in a dictionary first
```

Comment:
Let's make an internal task instead and remove the TODO. This will easily get lost here forever.